### PR TITLE
refactor(ui5-label): wrap text by default

### PR DESCRIPTION
**Pull request by dimovpetar**
The text of `ui5-label` now wraps by default.

BREAKING CHANGE: `wrapping-type` property default value has changed from `None` to `Normal`.
Before: 
```html
<ui5-label>some very very very long label</ui5-label> <!-- would truncate the text if there is not enough space -->
```

Now:
```html
<ui5-label>some very very very long label</ui5-title> <!-- would let the text wrap if there is not enough space -->
```

Related to https://github.com/SAP/ui5-webcomponents/issues/8461

### CODE REVIEW

**Comment by alexandar-mitsev** [comment]
LGTM , waiting for UA approval

File: packages/main/test/pages/Kitchen.html
Comment (BorisDafov) [inline packages/main/test/pages/Kitchen.html:505]: ..., but it's truncated

File: packages/main/test/pages/Kitchen.openui5.html
Comment (BorisDafov) [inline packages/main/test/pages/Kitchen.openui5.html:438]: , but this one is truncated

File: packages/website/docs/_components_pages/main/Label.mdx
Comment (BorisDafov) [inline packages/website/docs/_components_pages/main/Label.mdx:20]: To truncate it - set ...

File: packages/website/docs/_components_pages/main/Label.mdx
Comment (BorisDafov) [inline packages/website/docs/_components_pages/main/Label.mdx:25]: labels (also remove the apostrophe)

File: packages/website/docs/_samples/main/Label/TextWrapping/sample.html
Comment (BorisDafov) [inline packages/website/docs/_samples/main/Label/TextWrapping/sample.html:17]: will be truncated

File: packages/website/docs/_components_pages/main/Label.mdx
Comment (BorisDafov) [inline packages/website/docs/_components_pages/main/Label.mdx:30]: labels

File: packages/website/docs/_components_pages/main/Label.mdx
Comment (BorisDafov) [inline packages/website/docs/_components_pages/main/Label.mdx:20]: better:
The ui5-label ...

**Review by BorisDafov — CHANGES_REQUESTED** [review]

File: packages/main/test/pages/Kitchen.openui5.html
Comment (dimovpetar) [reply packages/main/test/pages/Kitchen.openui5.html]: Done

File: packages/main/test/pages/Kitchen.html
Comment (dimovpetar) [reply packages/main/test/pages/Kitchen.html]: Done

**Review by BorisDafov — COMMENTED** [review]
I approve the PR from UA perspective.

**Review by alexandar-mitsev — APPROVED** [review]

**Pull request merged**
